### PR TITLE
scrollIntoView: Skip overflow:hidden elements

### LINKED ISSUE #5355: Text layer moves up during search
1. Visit the following PDF (http://movelex.hu/fenyoapa/tori_6_tk_webre.pdf_150.pdf) with the text layer enabled, e.g. using the URL parameter:`http://movelex.hu/fenyoapa/tori_6_tk_webre.pdf_150.pdf#textLayer=visible`
2. Look at the text below the top-left image (looks all right).  
 pdfjs-textlayer
3. Use the search box and search for "pesti".
4. Look at the text below the top-left image, and observe that the text layer has shifted up.  
  pdfjs-textlayer-search

I've found this bug and PDF at https://stackoverflow.com/questions/25922247/how-to-fix-pdf-js-find-highlight-error-or-what-causes-it.

### PR DESCRIPTION
**Pull request by Rob--W**
Fixes #5355

### CODE REVIEW

File: web/ui_utils.js [hunk 1]
```diff
@@ -108,13 +108,14 @@ function scrollIntoView(element, spot) {
...
-  var offsetX = element.offsetLeft + element.clientLeft;
   if (!parent) {
     console.error('offsetParent is not set -- cannot scroll');
     return;
   }
-  while (parent.clientHeight === parent.scrollHeight) {
+  var offsetY = element.offsetTop + element.clientTop;
+  var offsetX = element.offsetLeft + element.clientLeft;
+  while (parent.clientHeight === parent.scrollHeight ||
+      getComputedStyle(parent).overflow === 'hidden') {
```
Comment (timvandermeij) [inline web/ui_utils.js]: Nit: please indent this with three more spaces to make it align with `parent`

**Comment by timvandermeij** [comment]
Seems to work just fine. After the nit is addressed, I'm fine with this solution. I'll leave the final review for @brendandahl once the nit is addressed.

File: web/ui_utils.js (hunk 1 above)
Comment (Rob--W) [inline web/ui_utils.js]: Done.

**Comment by timvandermeij** [comment]
@brendandahl Looks good to me. Could you do the final review?

**Pull request merged**